### PR TITLE
Feat/ballot history

### PR DESCRIPTION
**Pull request by scottPlusPlus**
Adds "history" to the Ballot object.
Changes the ballot date to be server-authoritative
Updates vote request to return the ballot with its id

Tested locally:

firefox_bWOE13E1vr
firefox_JoMMiIJXJl

NOTE:  I will suggest changing the ballot ID to be random, so voters don't know which order they voted in.  That may require a DB migration, so we may want to collect all the migrations we want to do and then organize that.

### CODE REVIEW

File: backend/src/Models/Ballots.ts [hunk 1]
```diff
@@ -34,20 +34,27 @@ class BallotsDB {
...
             ip_address      VARCHAR, 
-            votes           json NOT NULL
+            votes           json NOT NULL,
+            history         json
           );
         `;
         console.log(query);
         var p = this._postgresClient.query(query);
         return p.then((_: any) => {
+            return 'foo';
```
Comment (ArendPeter) [inline backend/src/Models/Ballots.ts:44]: Looks like this was some debug code that was left in?

**Review by ArendPeter — APPROVED** [review]

**Comment by ArendPeter** [comment]
Random ids sound good for the ballots

Regarding exposing the order, this could come up again if we add hash chaining down the road (discussed a bit here https://github.com/Equal-Vote/star-server/issues/63 ). Maybe enabling the hash chain feature would also require that the election admin enables other features to anonymize the votes?

**Comment by mikefranze** [comment]
I wouldn't worry about DB migrations, we don't have any important data yet so we can reset the database often.

File: backend/src/Models/Ballots.ts (hunk 1 above)
Comment (scottPlusPlus) [reply backend/src/Models/Ballots.ts:44]: Lol whoops, will fix.  Great catch!

**Pull request merged**